apps/frontend/src/app/components/user-search/user-search-result.component.scss

```diff
@@ -0,0 +1,24 @@
+.user-search-result {
+  display: flex;
+  align-items: center;
+
+  & .user-search-result__avatar {
+    width: 40px;
+    height: 40px;
+    border-radius: 5px;
+  }
+
+  & .user-search-result__content {
+    margin-left: 1rem;
+
+    & .content__alias {
+      font-size: 1.1rem;
+      font-weight: 700;
+    }
+
+    & .content__steamid {
+      font-size: 0.95rem;
+      font-style: italic;
+    }
+  }
+}
```

Comment:
Ampersand parent selectors are unnecessary here, Sass inserts them automatically. Though I guess you can argue it's clearer with them, so this is okay.

Generally we don't use BEM notation for Angular, since any CSS in a component's stylesheet is just scoped to that component. So whilst I'm a fan of BEM for projects where name collisions are a concern (we use it pretty religiously for our game UI), in this case it just means having super long class names for no reason. So essentially here you can just get rid of the block sections in the classnames e.g. 
```scss
.user-search-result {
  display: flex;
  align-items: center;

  .avatar {
    width: 40px;
    height: 40px;
    border-radius: 5px;
  }
...
}
```

Reply:
That's corrected 👍🏼 I didn't know it was scoped to the local component so yeah it kinda makes BEM redundant